fabric_plugin/tunnel.py

```diff
@@ -22,14 +22,11 @@ def documented_contextmanager(func):
 
 
 @documented_contextmanager
-def remote(remote_port, local_port=None, local_host="localhost",
+def remote(local_port, remote_port=0, local_host="localhost",
```

Comment:
I might be nitpicking.. but considering that `request_port_forward` returns an allocated port if port == 0, is there really a reason to allow to pass it as an argument to the function? The only reason might be if a user will need to modify the fabric plugin to allow passing a remote port because they have a policy not allowing to allocate random ports for the tunnel.. but if we want to allow users to do that, we should probably expose it in the blueprint.